src/index.css

```diff
@@ -6,4 +6,8 @@
   body {
     @apply font-poppins;
   }
+
+  .service-title {
```

Comment:
Please avoid using normal CSS directly. Our project is entirely based on Tailwind CSS. Only use regular CSS if Tailwind CSS cannot fulfill the styling requirements. Let's keep our styling consistent and maintainable.

Reply:
Not used however I have removed it